ndcube/ndcube.py

```diff
@@ -327,6 +303,10 @@ def axis_world_coords(self, *axes, edges=False):
                           for axis in new_int_axes]
         n_dependent_axes = [len(da) for da in dependent_axes]
 
+        # Store the lat/lon when skycoord=False
+        TS = None
```

Comment:
what is `TS`? Can we use a more descriptive name?

Reply:
Umm, that is for `Tx/Ty` of `skycoord`. The name can be improved